### DISCUSSION TITLE
Z2M taking control over existing devices

### DISCUSSION THREAD

**Discussion by enmasotsu** [Koenkk/zigbee2mqtt#11850] [Devices]
Hey there,

i am currently looking into moving from Deconz to z2m. I already installed z2m in a docker-container and connceted a sonoff ZB Bridge. It is running well, or maybe too good... 

In thew z2m configuration i wrote: permit_join: false
But there are devices joining nevertheless. Right now, my printer and my mini-fridge (currently connected by deconz in HomeAssistant) are turning on and off every 1 or 2 minutes... Very anoying. 

My question: How is it possible, that z2m is "taking control" over my existing and working devices?
I would like to build up multiple zigbee-networks (i got my reasons...) and z2m not to interfere with existing devises. What did i do wrong?

**Comment by Roxxor91** [comment]
Are you using the same adapter? You would also need to use 2 different adapters.

**Reply by enmasotsu** [reply]
One Adapter is Deconz-USB-Stick and the other Adapter is the ZB Bridge, those are completly seperated.
